training_methods/automl_training_method.py

```diff
@@ -0,0 +1,440 @@
+from tokenize import Double
+from typing import Any, Dict, List, Optional
+import pandas as pd
+import numpy as np
+
+from google.cloud import aiplatform, bigquery
+from google.cloud.aiplatform import models
+
+import utils
+from models import dataset, forecast_job_request
+from training_methods import training_method
+import constants
+
+
+TIME_COLUMN_PARAMETER = "timeColumn"
+TARGET_COLUMN_PARAMETER = "targetColumn"
+TIME_SERIES_IDENTIFIER_COLUMN_PARAMETER = "timeSeriesIdentifierColumn"
+FORECAST_HORIZON_PARAMETER = "forecastHorizon"
+DATA_GRANULARITY_UNIT_PARAMETER = "dataGranularityUnit"
+DATA_GRANULARITY_COUNT_PARAMETER = "dataGranularityCount"
+OPTIMIZATION_OBJECTIVE_PARAMETER = "optimizationObjective"
+COLUMN_SPECS_PARAMETER = "columnSpecs"
+TIME_SERIES_ATTRIBUTE_COLUMNS_PARAMETER = "timeSeriesAttributeColumns"
+CONTEXT_WINDOW_PARAMETER = "contextWindow"
+
+aiplatform.init()
```

Comment:
Maybe move to the `init` method of `AutoMLForecastingTrainingMethod`

Reply:
Created __init__ method and moved it there!